### PR TITLE
[MU4] Fix #322759: Inputing Ottava Bassa via menu or shortcut shows it above staff

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves: https://musescore.org/en/node/322759

This is for master what #8515 is for 3.x

### CODE REVIEW

File: src/engraving/libmscore/edit.cpp [hunk 1]
```diff
@@ -1869,6 +1869,10 @@ void Score::cmdAddOttava(OttavaType type)
             }
             Ottava* ottava = new Ottava(this);
             ottava->setOttavaType(type);
+            if (type == OttavaType::OTTAVA_8VB/* || type == OttavaType::OTTAVA_15MB || type == OttavaType::OTTAVA_22MB*/) {
```
Comment (Jojo-Schmitz) [inline src/engraving/libmscore/edit.cpp:1872]: No need to check for `OttavaType::OTTAVA_15MB` and `OttavaType::OTTAVA_22MB`, as there's no shortcut for those

File: src/engraving/libmscore/edit.cpp [hunk 2]
```diff
@@ -1888,7 +1892,10 @@ void Score::cmdAddOttava(OttavaType type)
 
         Ottava* ottava = new Ottava(this);
         ottava->setOttavaType(type);
-
+        if (type == OttavaType::OTTAVA_8VB/* || type == OttavaType::OTTAVA_15MB || type == OttavaType::OTTAVA_22MB*/) {
```
Comment (Jojo-Schmitz) [inline src/engraving/libmscore/edit.cpp:1895]: same

File: src/palette/internal/palette/palettecreator.cpp [hunk 3]
```diff
@@ -1423,6 +1423,7 @@ PalettePanel* PaletteCreator::newLinesPalettePanel()
 
     ottava = makeElement<Ottava>(gscore);
     ottava->setOttavaType(OttavaType::OTTAVA_22MB);
+    ottava->setPlacement(Placement::BELOW);
```
Comment (Jojo-Schmitz) [inline src/palette/internal/palette/palettecreator.cpp:1426]: May not be needed, certainly not for the issue at hand, but I found this to be pretty inconsistent, at least.

File: src/engraving/libmscore/edit.cpp (hunk 1 above)
Comment (wizofaus) [reply src/engraving/libmscore/edit.cpp]: But logically the user should be able to define such shortcuts ( and you probably can by editing files directly), so I'd think we should support that anyway?
More importantly, shouldn't this logic belong inside Ottava itself? (i.e. setOttavaType( ) should set the "placement" property as appropriate)

File: src/engraving/libmscore/edit.cpp (hunk 2 above)
Comment (wizofaus) [reply src/engraving/libmscore/edit.cpp]: Code duplication, again, would make sense to me to keep inside Ottava::setOttavaType

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/edit.cpp]: That duplication is not my fault though, and I don't plan to fix that, not for this issue.

File: src/engraving/libmscore/edit.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [reply src/engraving/libmscore/edit.cpp]: I don't think you can just invent new shortcuts without code changes. Directly editing the shortcuts.xml won't work, as far as I can tell.
Yes. that decision, above/below, might better be taken elsewhere, something for another PT maybe, here I just tried to keep the changes as minimal as possible

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/edit.cpp]: Hmm, maybe that `cmdAddOttava()` can get used from whithin plugins, and then with arguments other than just `OttavaType::OTTAVA_8VA` and `OttavaType::OTTAVA_8VB`.

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/edit.cpp]: Edit: no, It can't, only the  "add-8va" and "add-8vb" from cmd.cpp

**Pull request merged**